daisy_workflows/image_build/debian/debian_10_fai.wf.json

```diff
@@ -30,6 +33,33 @@
           "google_cloud_repo": "${google_cloud_repo}"
         }
       }
+    },
+    "copy-gcs-object": {
```

Comment:
This isn't doing anything. We still need a way to specify where the tar ends up.

Reply:
this is a necessary step because of how daisy model validation works. by adding this step, which doesn't validate its inputs, we can get daisy to assume the tarball will exist.

Reply:
Yes, this is to bypass the validation of bucket object in daisy.